hello_world_on_chain/test/user.ts

```diff
@@ -57,23 +57,29 @@ const loadUserContractABI = () => {
     }
 }
 
+interface UserContract extends ethers.Contract {
```

Comment:
@mwittie can you see if this update addresses the warnings that you are getting on your system in the typescript files and if not, can you fix?  I am not sure what tools your Ide is using that is generating the notifications and so I can't reproduce.  Out of scope for this PR, but, it could be worth making some decisions on how do you want to handle things like linting and code checks for this repo. Note that in https://blocky.atlassian.net/browse/BKY-6457 I am just setting up workflows and testing.

Reply:
confirmed I'm not getting IDE warnings